website/content/docs/agent/config/config-files.mdx

```diff
@@ -2220,3 +2218,7 @@ server.
     The default value is `250`. It is based on a load test of 5,000 streams connected to a single server with two CPU cores.
 
     If necessary, you can lower or increase the limit without a rolling restart by using the `consul reload` command or by sending the server a `SIGHUP`.
+
+<!-- list of reference-style links -->
```

Comment:
Why is this included?

Reply:
Oh, this fixes broken links I noticed. There are a bunch of places on this page where `[go-sockaddr]` is being rendered literally in the HTML:

<img width="854" alt="Screen Shot 2022-12-07 at 4 42 18 PM" src="https://user-images.githubusercontent.com/1077740/206312666-03e0c8a6-7d90-4ba0-8944-82f861241d4f.png">

Although, I should make this doc fix a separate PR and backport it. I'll do that instead

Reply:
Ok, moved this fix to https://github.com/hashicorp/consul/pull/15723